src/main/java/frc/team2412/robot/subsystems/DrivebaseSubsystem.java

```diff
@@ -167,15 +208,15 @@ public void drive(
 			boolean autoBalance) {
 		// Auto balancing will only be used in autonomous
```

Comment:
I hope we also get to use it during teleop, as the computer will be way faster at balancing than a human.

Reply:
Holding down a bumper, trigger, or button during teleop might be an option.  Kind of like automatic parallel parking.
It would allow computer to stop when balanced or the driver to stop if it is too close to the edge or similar.

Reply:
Added in #34